modules/channel-web/src/views/web/index.jsx

```diff
@@ -279,23 +281,51 @@ export default class Web extends React.Component {
     this.props.bp.events.on('guest.webchat.typing', this.handleBotTyping)
   }
 
+  fetchBot = () => {
+    const axios = this.props.bp.axios
+    const url = '/'
+    return axios.get(url, this.axiosConfig).then(({ data }) => this.setState({ bot: data }))
+  }
+
   checkForExpiredExternalToken = error => {
     if (_.get(error, 'response.data.errorCode') === 'BP_0401') {
       this.setState({ config: { ...this.state.config, externalAuthToken: undefined } }, this.updateAxiosConfig)
       console.log(`External token expired or invalid. Removed from future requests`)
     }
   }
 
+  sendUserVisit = () => {
+    this.handleSendData({
+      type: 'visit',
+      text: 'User visit',
+      timezone: moment().utcOffset() / 60
+    })
+      .then(res => {
+        if (res.data.conversationId !== null) {
+          this.setState({ currentConversationId: res.data.conversationId }, async () => {
+            await this.fetchCurrentConversation(res.data.conversationId)
+          })
+        }
+      })
+      .catch(this.checkForExpiredExternalToken)
+  }
+
+  sendGetStarted = () => {
+    // TODO: Add && showGetStarted is true
+    console.log('Get Started', this.state)
```

Comment:
Do we need this `console.log()`?

Reply:
I think @rndlaine opened this PR with no intention to merge at this moment. Notice that there's still a TODO list in the PR description. That means he opened  PR so we can get our minds around what he's doing and how he intent to complet the work (since the feature needs lots of new code)